### ISSUE TITLE
Problem using custom version

### ISSUE THREAD

**Issue by alexz707** [phpmetrics/PhpMetrics-jetbrains#14]
Hi!
When I want to use a newer version of phpmetrics I get a blank webpage when analysing.
If i switch back to the included version it works.
how can I upgrade the phpmetrics version?

phpstorm 2016.3.2
latest jar
phpmetrics built in 1.8.3
tried v2.0.0-rc (and the untagged versions)

it still works with v1.10.0

can u please have a look at it? 

thanks!

**Comment by UFOMelkor** [comment]
Yes, PhpMetrics v2.0 does not work actually. While the support of every version < 2 is made of one single file, v2.0 uses a full directory for the report. I will have a look at it.

@phpmetrics/release-masters Do you think we can raise the [minimal required SDK to 7](http://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createTempDirectory%28java.nio.file.Path,%20java.lang.String,%20java.nio.file.attribute.FileAttribute...%29)?

**Comment by panosru** [comment]
any news on that one?

Thanks
